src/groundStation.py

```diff
@@ -24,16 +24,23 @@
 from system import SatelliteNodes
 
 class GroundStation:
+    """This is the base class that CLI extends.
+    Attributes:
+        networkManager: an object that uses the CubeSat Protocol to establish a network connection
+        interactive: an object that handles the keyboard input
+        inputHandler: an object that logs in the keyboard input
+    """
     def __init__(self, opts):
         hkeyfile = open(opts.hkeyfile, "rb")
         hkey = hkeyfile.read()
         xkeyfile = open(opts.xkeyfile, "rb")
         xkey = xkeyfile.read()
         addr = 0
         nodeType = "UHF"
+        # GroundNodes = [("GND", "UHF", 16), ("GND", "SBAND", 17), ("GND", "PIPE", 24), ("GND", "BEACON", 99)]
         for i in GroundNodes:
             if i[1] == nodeType:
-                addr = i[2]
+                addr = i[2]         # Q: so why not just set addr to 16?
```

Comment:
This was meant to be extended to allow being other types of ground node but we never got around to it

Reply:
Okay 👍